### ISSUE TITLE
Error in rbindlist whilst running getPartisAnnotations

### ISSUE THREAD

**Issue by Pezhvuk** [matsengrp/sumrep#26]
Hi Branden,

While I have successfully run a full sumrep-partis analysis on some files, the exact following error keeps recurring for various files from different datasets (with each file varying in size), when running `getPartisAnnotations`.

```
Error in rbindlist(l, use.names, fill, idcol) : 
  Item 2 has 33 columns, inconsistent with item 1 which has 36 columns. If instead you need to fill missing columns, use set argument 'fill' to TRUE.
Calls: getPartisAnnotations ... eval -> eval -> rbind -> rbind -> <Anonymous> -> rbindlist
```
I cannot find any reference to it in the repo, so, any advice would be very appreciated.

Cheers,
Peji.

**Comment by Pezhvuk** [comment]
potential suspect:

https://github.com/matsengrp/sumrep/blob/127344a7a72a0afdbc7330b40ed5264099d3ddbb/R/PartisFunctions.R#L261-L263

**Comment by Pezhvuk** [comment]
changing the  line referenced above to `collapsed_dat <- rbind(collapsed_dat, clone_dat, fill=TRUE)` seems to have fixed the problem, but now I'm getting another error:

`Error: `data` must be uniquely named but has duplicate elements`

**Comment by Pezhvuk** [comment]
Hi @BrandenOlson,

Given that you were not experiencing this issue, and updating Partis resolved the follow up problem to this in #28, I suspect this issue is perhaps also caused due to outdated Partis, and close this issue.
